### DISCUSSION TITLE
Dualshock 3 Random Stick Movements But Only in Some Games

### DISCUSSION THREAD

**Discussion by BakaSolaire** [nefarius/DsHidMini#192] [Support]
For some reason this problem doesn't always exist. For example the soft works absolutely fine in Yakuza 6 but if I run Nioh 2 or Skul: The Hero Slayer my character starts walking North-West constantly. This is not stick drifting because it happens only in some games which is really strange but in makes them unplayable, sadly.

**Comment by nefarius** [comment]
Try different modes.

**Reply by BakaSolaire** [reply]
didn't help unfortunately

**Reply by nefarius** [reply]
Well, the driver doesn't come up with random movements on its own so it's either a layout or a hardware issue, both things I can not help with as it's not within my reach to fix in software.

**Reply by EZE404** [reply]
well, I use DS4Window with that mode selected and it happens to me sometimes while playing burnout paradise. But that game is so buggy that I thought it was the game.

**Reply by nefarius** [reply]
Not a driver issues. Driver translates what it gets from hardware. As simple as that.

**Comment by Yurnerosk** [comment]
Happened the same, but with rocket league and Street Fighter 5. This would happen a lot back in SCPToolkit, but I've never found a solution. Weirdly enough, the consecutive North-West inputs happen periodically, even with different DS3 controllers

**Reply by Yurnerosk** [reply]
Also, I don't mean to cry that the software is faulty. I just think that's not controller hardware problem. So maybe it is usb interface things?
